### PR TITLE
done with the landing page of component UI

### PR DESCRIPTION
**Pull request by ajit-sharmaGH**

### CODE REVIEW

**Comment by krishnakant01** [comment]
Hey, I have gone through your code. It looks good to merge.

File: CSS/base/color.css [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    --dark-tertiary: #282828;
+    --dark-background: #282828;
+    --dark-primary-text: #b3b3b3;
+    --dark-bold-text: #ffffff;
+    --pink: #ffc0cb;
+    --yellow: #ffff00;
+    --white: #ffffff;
+    --black: #000;
+    --red: red;
+    --green: green;
```
Comment (Mohasin-Haque) [inline CSS/base/color.css:19]: You can use purpose here.. if you have to use the name of color then you can do it directly..

File: CSS/base/nav.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
+nav {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    /* min-height: 8vh; */
```
Comment (Mohasin-Haque) [inline CSS/base/nav.css:5]: remove the commented part

File: CSS/base/nav.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+        width: 60%;
+    }
+}
+
+@media screen and (max-width:700px) {
+    body {
+        overflow-x: hidden;
+    }
+    .nav-links {
+        position: absolute;
```
Comment (Mohasin-Haque) [inline CSS/base/nav.css:85]: Use media queries at last of the page.

File: CSS/base/nav.css [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+        top: 0px;
+        height: 16vh;
+        top: -15vh;
+        background-color: grey;
+        border-radius: 1rem;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
```
Comment (Mohasin-Haque) [inline CSS/base/nav.css:94]: you can make a separate class for flex instead of using it again and again.

File: CSS/base/reset.css [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+/* Form */
+
+button,
+input,
+select,
+textarea {
+    margin: 0;
+}
+
+
```
Comment (Mohasin-Haque) [inline CSS/base/reset.css:61]: please run prettier and also do take care of the indentation.

File: Htmlfiles/documentation.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Mohasin-Haque) [inline Htmlfiles/documentation.html:7]: you can add more meta description to personalise the ui.

File: Htmlfiles/documentation.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+            <li>
+                <a href="https://github.com/ajit-sharmaGH">Github</a>
+            </li>
+
+        </ul>
+        <div class="burger">
+            <div class="line1"></div>
+            <div class="line2"></div>
+            <div class="line3"></div>
+
```
Comment (Mohasin-Haque) [inline Htmlfiles/documentation.html:35]: why gap?

File: Htmlfiles/install.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Installation Process</title>
+    <link href="../../CSS/install.css" rel="stylesheet" />
+</head>
+
+<body>
+
+    <nav>
+        <div class="logo">
```
Comment (Mohasin-Haque) [inline Htmlfiles/install.html:15]: formate the file

File: style.css [hunk 9]
```diff
@@ -0,0 +1,120 @@
+@import url("CSS/base/reset.css");
+@import url("CSS/base/color.css");
+@import url("CSS/base/nav.css");
```
Comment (Mohasin-Haque) [inline style.css:3]: you can add comments to sanity

**Review by Mohasin-Haque — CHANGES_REQUESTED** [review]
Your code is good but needs some changes as commented.. please go through your files once.

File: Htmlfiles/install.html (hunk 8 above)
Comment (ajit-sharmaGH) [reply Htmlfiles/install.html]: Okk will do that

**Pull request merged**